.travis.yml

```diff
@@ -1,28 +1,54 @@
-dist: trusty
+dist: xenial
 sudo: false
 language: rust
-cache:
-  directories:
-    - $HOME/.cargo
-    - $HOME/.cache
-    - $TRAVIS_BUILD_DIR/target
-before_cache:
```

Comment:
The step is necessary to keep cache small.

Reply:
even on windows platform ? maybe i need more test tonight